src/_layouts/components/navigation.11ty.js

```diff
@@ -1,26 +1,59 @@
 exports.getNavigation = (eleventy, langCode, objectId, type) => {
   const config = eleventy.getConfig();
-  const serachSuffix = type === 'literature' ? 'kind=literature_references' : 'kind=works';
+  
+  /* Nav Links ----------------------------------------------------------------- */
+
+  const baseUrlHomepage = config.cranachBaseUrlHomepage[langCode];
+  const toCdaHomepage = `
+    <a class="button button--is-transparent js-go-to-search cda-logo" href="${baseUrlHomepage}">
+      <span class="button__text button__text--is-important">cda_</span>
+    </a>
+  `;
+
   const cranachSearchURL = `${config.cranachSearchURL[process.env.ELEVENTY_ENV].replace(/langCode/, langCode)}?loadLatestSearchConfiguration=true`;
+  const toMainSearch = `
+    <a class="button button--is-transparent js-go-to-search" href="${cranachSearchURL}">
+      <span class="button__icon button__icon--is-large icon has-interaction">apps</span>
+      <span class="button__text button__text--is-important">${eleventy.translate('zur-werksuche', langCode)}</span>
+    </a>
+  `;
+  
+  const cranachSearchUrlLiterature = `${config.cranachSearchURL[process.env.ELEVENTY_ENV].replace(/langCode/, langCode)}?kind=literature_references&oadLatestSearchConfiguration=true`;
```

Comment:
Hier ist ein kleiner typo der dafür sorgt, dass die letzten Search-Konfiguration. Werde ich gleich in einem commit fixen.

`&oadLatestSearchConfiguration=true`